jonathaneeckhout/jdungeon
# feat(debug_mute): add variable to .env to start client with sound muted

Fixes #45 

Testing Scenario:
- Add `AUDIO_MUTE=true` to your `.env` file and the client should start with sound muted
- Add `AUDIO_MUTE=false` to your `.env` file and client starts with audio on
- Remove the configuration `AUDIO_MUTE` from your .env file. It should default to having audio on
